index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/04970764d6.js" crossorigin="anonymous"></script>
+</head>
+<body>
+        <header class="header">
+            <p class="logo">My Logo</p>
+            <nav class="menu"><img src="image/Union.svg" alt="Menu"></nav>
+        </header>
+        <main>
+            <section class="background">
+        <h2 class="head-title">I’m Mariane <br>
+            Glad to see you!</h2>
```

Comment:
- [ ] We use the `h1` tag for the main title. In your project, the ` I’m Mariane, Glad to see you!` is the main title of your portfolio. Please, Kindly change it.